source/guides/stacks/dlrs-inference.rst

```diff
@@ -0,0 +1,1035 @@
+.. _dlrs-inference:
+
+AI Inference with the Deep Learning Reference Stack
+###################################################
+
+In this guide we explore a solution for using the Deep Learning Reference Stack with a Seldon Core\* platform that has been deployed on Kubernetes\*.
+
+.. contents::
+   :local:
+   :depth: 2
+
+Overview
+********
+
+The solution covered here uses the following software components:
+
+* `Deep Learning Reference Stack`_ which is a |CL-ATTR| based Docker\* container providing deep learning frameworks and is optimized for Intel Xeon Scalable platforms.
+* `Kubeflow`_ is the machine learning toolkit for Kubernetes that helps with deployment of Seldon Core and Istio components.
+* `Seldon Core`_ is a software platform for deploying machine learning models. We use the DLRS container to serve the OpenVino\* framework for inference with the Seldon Core.
+* The OpenVino Model Server is included in DLRS and provides the OpenVino framework for inference. From OpenVino, the `OpenVino Toolkit`_ provides improved neural network performance on a variety of Intel processors. For this guide, we converted pre-trained Caffe models into the `Intermediate Representation(IR)`_ of ResNet50 with the OpenVino toolkit.
+* `Istio`_ is a traffic manager and performs the load balancing for service requests in the cluster.
+* Pre-processing container converts jpeg content into NumPy array
+* Post-processing container converts an array of classification probabilities to a human-readable class name.
+* Pre and post-processing containers are created using the `Source-to-Image`_ (S2I)toolkit, which builds reproducible container images from source code/
+* `Min.io`_ is used as the distributed object storage for models
+
+.. figure:: /_figures/stacks/kubeflow-seldon-dlrs-example-diagram.png
+   :alt: Example diagram with DLRS deployed by Seldon
+   :width:     600
+
+Prerequisites
+*************
+
+Although this guide assumes a |CL| host system, it has also been validated with the following software components.
+
+.. list-table:: **Table 1. Software Component Versions**
+   :widths: 16,16
+   :header-rows: 1
+
+   * - Component
+     - Version
+
+   * - DLRS
+     - 0.4.0
+
+   * - Docker
+     - 18.09
+
+   * - Kubernetes
+     - 1.15.3
+
+   * - Source-to-Image
+     - 1.1.14
+
+   * - Helm
+     - 2.14.3
+
+   * - Kubeflow
+     - 0.6.1
+
+   * - Seldon
+     - 0.3.2
+
+   * - Rook
+     - 1.0.5
+
+   * - Ceph
+     - 14.2.1-20190430
+
+   * - Minio
+     - RELEASE.2019-04-23T23-50-36Z
+
+   * - CentOS
+     - 7.6
+
+   * - OpenVINO Toolkit
+     - 2019 R1.0.1
+
+   * - MKL-DNN
+     - 0.19
+
+Recommended Hardware
+====================
+
+We validated this guide on an `Intel Cascade Lake`_ server and this is recommended to get optimal performance and take advantage of the built in  Intel® Deep Learning Boost functionality.
+
+Required Software
+=================
+
+* :ref:`Install <bare-metal-install-desktop>` |CL| on your host system
```

Comment:
Could you combine these bullets and the actual commands? This would avoid repetition. 
For example, after `bare-metal-install-desktop`, this could be revised as:
```
.. code-block:: bash
   sudo swupd bundle-add containers-basic cloud-native-basic
```

Reply:
latest commit addresses this now.